### PR TITLE
Add python 3.4 support

### LINKED ISSUE #77: python3 support
I am totally new to python. From issue 70 (syntax error), it appears that this package does not support python 3. Why not? Thanks.

### PR DESCRIPTION
**Pull request by Darune**
Support both python 2.7 and python 3.4

Fix #77

### CODE REVIEW

File: linkedin/utils.py [hunk 1]
```diff
@@ -16,6 +13,22 @@
         import json
 
 
+if sys.version < '3':
```
Comment (salty-horse) [inline linkedin/utils.py]: Isn't it better to use ``sys.version_info < (3,)``? (Not that Python 10 is on the horizon)

**Comment by salty-horse** [comment]
The classifiers in setup.py should be updated to list the supported Python 3 versions:
```python
'Programming Language :: Python :: 3',
'Programming Language :: Python :: 3.4'
```

**Comment by Darune** [comment]
Thank you salty horse, i've added your suggestions and push with --force flag to update this pull request

**Comment by tinkhine** [comment]
Is this pull request stable? When is it planning to merge? Thanks.

**Comment by Darune** [comment]
This pull request is stable, however it doesn't solve #78 which i plan to solve later

**Comment by pquentin** [comment]
@ozgur any news on this? Can you please consider this pull request? This could pave the way to other pull requests to support the newer API versions.

**Comment by ozgur** [comment]
:100:

**Comment by pquentin** [comment]
:clap:

**Pull request merged**